django/core/cache/backends/memcached.py

```diff
@@ -154,7 +154,7 @@ class MemcachedCache(BaseMemcachedCache):
     "An implementation of a cache binding using python-memcached"
     def __init__(self, server, params):
         import memcache
-        super(MemcachedCache, self).__init__(server, params,
+        super().__init__(server, params,
```

Comment:
And also fix formatting in cases like this.  Use hanging indent::
```
super().__init__(
    server, param, library=pylibmc, value_not_found_exception=pylibmc.NotFound,
)
```
or a single line if it fits in 119 characters.